packtools/sps/validation/article_toc_sections.py

```diff
@@ -0,0 +1,82 @@
+from packtools.sps.models.article_toc_sections import ArticleTocSections
+from packtools.sps.models.article_titles import ArticleTitles
+
+
+class ArticleTocSectionsValidation:
+    def __init__(self, xmltree):
+        self.xmltree = xmltree
+        self.article_toc_sections = ArticleTocSections(xmltree)
+        self.article_titles = ArticleTitles(xmltree)
+
+    def validate_article_toc_sections(self, article_section_titles):
+        resp = []
+        number_of_titles_found = len(self.article_toc_sections.article_section_dict) + \
+                                 len(self.article_toc_sections.sub_article_section_dict)
+        number_of_titles_expected = len(article_section_titles)
+        if number_of_titles_found != number_of_titles_expected:
+            resp.append(
+                dict(
+                    object='section title',
+                    expected_value=f"{number_of_titles_expected} section titles",
+                    obtained_value=f"{number_of_titles_found} section titles",
+                    result=False,
+                    message="ERROR, number of titles found is different from the expected number of titles"
+                )
+            )
+        else:
+            for lang, text in self.article_toc_sections.article_section_dict.items():
+                if text == article_section_titles.get(lang):
```

Comment:
@Rossi-Luciano então aqui deve ficar `text in article_section_titles.get(lang)`, ou seja, trocar `==` por `in`.

Reply:
OK, @robertatakenaka